src/templates/_common/includes/google_optimizer.jsx

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
```

Comment:
why are we disabling eslint? please only disable the specific rule, on the specific line/smaller section that is effected

Reply:
I think we do the same in gtm.jsx. should be because of using `dangerouslySetInnerHTML` ?

Reply:
then we should only disable that specific rule that breaks